### PR TITLE
Add module logistics order donation

### PR DESCRIPTION
**Pull request by yvaucher**
This module provides a new type of logistic order for In-Kind Donation

It also re-factor logistic_order to add a type extendable selection field on sale order

### CODE REVIEW

**Comment by lepistone** [comment]
@yvaucher I think I know what the build error is. It looks like the problems we're having where in tests we have a partial visibility of other modules' fields.

**Comment by yvaucher** [comment]
@lepistone I fixed it, a change was missing in logistic_requisition

**Comment by lepistone** [comment]
thanks! :+1:

**Comment by yvaucher** [comment]
rebased

**Comment by jgrandguillaume** [comment]
:+1:  LGTM

File: logistic_order_donation/README.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Logistics Order - Donation
+==========================
+
+This module adds a type of Logisitcs Orders for In-Kind Donations
```
Comment (lepistone) [inline logistic_order_donation/README.rst]: typo

File: logistic_order_donation/README.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+A new menu entry permits to manage In-Kind Donations which is a new type of
+Logistics Order.
+
+This kind of Logistics Order use a specific route for In-Kind Donations which
```
Comment (lepistone) [inline logistic_order_donation/README.rst]: uses

File: logistic_order_donation/README.rst [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+transfer the goods from Donor location to Customer location as a Dropshipping
```
Comment (lepistone) [inline logistic_order_donation/README.rst]: transfers

File: logistic_order_donation/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+
+{"name": "Logistics Order - Donation",
+ "summary": "Sales Order as In-Kind Donations",
+ "version": "0.0.1",
```
Comment (lepistone) [inline logistic_order_donation/__openerp__.py]: Do we need all these zeroes? Probably `1.0` or `0.1` is enough.

File: logistic_order_donation/data/stock.xml [hunk 5]
```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<openerp>
+  <data>
+
+    <record id="stock_location_donors" model="stock.location">
+      <field name="name">Donors Locations</field>
+      <field name="usage">supplier</field>
+      <field name="company_id"></field>
```
Comment (lepistone) [inline logistic_order_donation/data/stock.xml]: `eval="False"` ?

**Comment by lepistone** [comment]
It's :+1: with a couple minor remarks. Ping me when you're ready for a merge, thanks!

**Comment by yvaucher** [comment]
@lepistone fixed

**Pull request merged**